packages/app/app/actions/importfavs.js

```diff
@@ -0,0 +1,99 @@
+import logger from 'electron-timber';
+import { LastFmApi } from '@nuclear/core';
+import { store } from '../persistence/store';
+import globals from '../globals';
+import * as FavoritesActions from './favorites';
+const lastfm = new LastFmApi(globals.lastfmApiKey, globals.lastfmApiSecret);
+
+export const FAV_IMPORT_INIT = 'FAV_IMPORT_INIT';
+
+export const LASTFM_FAV_IMPORT_START = 'LASTFM_FAV_IMPORT_START';
+export const LASTFM_FAV_IMPORT_SUCCESS_1 = 'LASTFM_FAV_IMPORT_SUCCESS_1';
+export const LASTFM_FAV_IMPORT_SUCCESS_FINAL = 'LASTFM_FAV_IMPORT_SUCCESS_FINAL';
+export const LASTFM_FAV_IMPORT_ERROR = 'LASTFM_FAV_IMPORT_ERROR';
+
+export function FavImportInit() {
+  return dispatch => {
+    dispatch({
+      type: FAV_IMPORT_INIT,
+      payload: {
+        lastFmFavImportStatus: true,
+        lastFmFavImportMessage: ''
+      }
+    });
+  };
+}
+
+function FmFavError() {
+  return {
+    type: LASTFM_FAV_IMPORT_ERROR,
+    payload: null
```

Comment:
You don't have to specify a payload if you don't want to pass anything.
This could be turned into a single line:
```javascript
const fmFavError = () => ({ type: LASTFM_FAV_IMPORT_ERROR })
```